app/src/main/java/com/github/libretube/adapters/CommentsAdapter.kt

```diff
@@ -0,0 +1,53 @@
+package com.github.libretube.adapters
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import com.github.libretube.R
+import com.github.libretube.formatShort
+import com.github.libretube.obj.Comment
+import com.github.libretube.obj.StreamItem
+import com.squareup.picasso.Picasso
+
+class CommentsAdapter(private val comments: MutableList<Comment>):  RecyclerView.Adapter<ViewHolder>(){
+
+    fun updateItems(newItems: List<Comment>){
+        comments.addAll(newItems)
+        notifyDataSetChanged()
```

Comment:
This can be optimized to notifyDataSet inserted with the old list's size for the index, and the size of the page.

Reply:
Could you just edit it yourself please?

Reply:
Changed it if that's what you've been talking about

Reply:
Feels like it's a lot faster now